HCCGo/app/js/jobHistoryCtrl.js

```diff
@@ -30,8 +30,8 @@ jobHistoryModule.service('jobService', function() {
 
   }
 
-  $scope.loadJob = function(job) {
-
+  $scope.loadJob = function(job, clone) {
+    job.clone = clone;
```

Comment:
I figured there was no use case where a user would want to create a clone just for the sake of having an exact duplicate in jobHistory without running it. So instead of creating a duplicate and then making them do the extra click to load that one, I thought it was better to just send them right to jobSubmission with the clone loaded.